manifest/manifest.yaml

```diff
@@ -123,6 +123,18 @@ components:
           type: boolean
           description: If true, the write object will inherit the mapping from the read object. If false, the write object will have no mapping.
           example: true
+        valueDefaults:
```

Comment:
because of the change in requirements (see Slack), can you revert the changes to manifest.yaml? The changes to config.yaml look good and can get merged.